templates/vue/src/components/Lightbox/media/TextMedia.vue

```diff
@@ -39,13 +39,15 @@ export default {
 
 <style lang="scss" scoped>
 .article {
+  color: #111;
```

Comment:
Again, not sure if we need this on a child element - can we keep it on the parent elements instead? Is there an overwrite somewhere that should be removed instead?

Reply:
yeah, turns out the rule isnt needed since it's inhereited.